response.go

```diff
@@ -175,3 +175,13 @@ type Log struct {
 	Removed         bool     `json:"removed"`
 }
 
+//GasPrices holds info for Gas Oracle queries
+//Gas Prices are returned in Gwei
+type GasPrices struct {
+	LastBlock            string
+	SafeGasPrice         string
```

Comment:
Since gas price is in Gwei, maybe we should use a `float64`?

I am curious, what data type are you using when you consume `GasPrices` struct? We may use the type dirctly so that our user don't have to do the convertion by themselves.

Reply:
I think having `LastBlock` be an int and converting the prices to either `float64` or `decimal.Decimal` is better. I personally would consume the Gas prices as `decimal.Decimal`, however, I understand not going that route since the idea behind this API is to leverage the go standard library.

Reply:
See [384329f](https://github.com/nanmu42/etherscan-api/pull/74/commits/384329f2261b7e66a1ab6dc175af1010a537a8e1)